frontend/src/lib/components/ProjectList.svelte

```diff
@@ -1,42 +1,70 @@
 <script lang="ts">
   import t from '$lib/i18n';
+  import Icon from '$lib/icons/Icon.svelte';
   import { Badge } from './Badges';
   import { getProjectTypeIcon } from './ProjectType';
-  import type { ProjectItem } from './Projects';
+  import type { ProjectItemWithDraftStatus } from './Projects';
 
-  export let projects: ProjectItem[];
+  export let projects: ProjectItemWithDraftStatus[];
 </script>
 
-<div class="grid grid-cols-2 sm:grid-cols-3 auto-rows-fr gap-2 md:gap-4">
+<div class="grid grid-cols-1 xs:grid-cols-2 sm:grid-cols-3 auto-rows-fr gap-2 md:gap-4 justify-items-center">
   {#each projects as project}
-    <a class="card aspect-square bg-base-200 shadow-base-300 group overflow-hidden" href={`/project/${project.code}`}>
-      <div class="bg" style="background-image: url('{getProjectTypeIcon(project.type)}')" />
-      <div class="card-body z-[1]">
-        <h2 class="card-title overflow-hidden text-ellipsis" title={project.name}>
-          <span class="text-primary inline-flex gap-2 items-center">
-            {project.name}
-          </span>
-        </h2>
-
-        <p>{project.code}</p>
-
-        <p>
-          <span class="i-mdi-account text-xl mb-[-4px]" /> {project.userCount}
-        </p>
-
-        <p class="flex items-end">
-          {#if project.lastCommit}
-            {$t('projectlist.last_change', {
-              lastChange: new Date(project.lastCommit),
+    {#if project.isDraft}
+      <div data-sveltekit-preload-data="false" class="draft card aspect-square bg-base-200 shadow-base-300 group overflow-hidden">
+        <div class="bg" style="background-image: url('{getProjectTypeIcon(project.type)}')" />
+        <div class="card-body z-[1]">
+          <h2 class="card-title overflow-hidden text-ellipsis" title={project.name}>
+            <span class="text-primary inline-flex gap-2 items-center">
+              {project.name}
+              <span
+                class="tooltip text-warning text-xl shrink-0 leading-0"
+                data-tip={$t('projectlist.is_draft')}>
+                <Icon icon="i-mdi-script" />
+              </span>
```

Comment:
The tooltip doesn't seem to work. I'm not sure why, but I don't think we need it, because we have the badge clearly labelling the project.
```suggestion
                <Icon icon="i-mdi-script" />
```

Reply:
hey @myieye 👋 do we want the scroll icon to be yellow? I am asking because in list view it's yellow, but it appears to be blue in grid view now.
![Screenshot 2024-06-12 132032](https://github.com/sillsdev/languageforge-lexbox/assets/30375118/c8268eb4-4ff1-4509-85d9-060376e14b30)

Reply:
Yeah, I think you should make it yellow 👍